server/src/controllers/sections/deleteSectionController.ts

```diff
@@ -0,0 +1,21 @@
+import { NextFunction, Request, Response } from 'express';
+import { deleteSectionQuery } from '../../database';
+
+const deleteSectionController = async (req: Request, res: Response, next: NextFunction) => {
+  const { id } = req.params;
```

Comment:
what if there is no section has this id ?

Reply:
If there is no section, you will not reach the button to delete it🙂